docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+### Is ngrok a dedicated instance or multi-tenant?
+
+ngrok is a multi-tenant application with services shared across our customer base, including your vendor and your site-to-site architecture.
+
+## Security
+
+### How can I ensure ngrok does not pose a security risk to my infrastructure?
+
+We recommend you begin by exploring our [Security, Privacy, and Compliance](https://ngrok.com/security) page, followed by our [Trust Center](https://trust.ngrok.com/).
+
+ngrok is deeply configurable to enforce your established security policies, including:
+
+- [Prevent unauthorized usage of ngrok](#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?) not outside of the site-to-site connectivity with your vendor.
+- Create a single account tenant, enforce [Account Domain Controls](https://ngrok.com/blog-post/account-domain-controls) to route all ngrok users with an email address `@your-company.com` into it, and enable role-based access controls to limit those accounts.
+- Apply [Access Control Lists](/docs/iam/rbac/#additional-restrictions-with-acls) (ACLs) to determine where ngrok agents can be created within your network.
+- Enable the [IP restrictions](/docs/http/traffic-policy/actions/restrict-ips/) action on your endpoint(s) to allow only trusted connections from your vendor.
+- Enforce basic, OAuth, OpenID, SAML, JWT, or mTLS authentication on your endpoint(s).
+
+### How does ngrok’s traffic encryption work? {#how-does-ngrok’s-traffic-encryption-work?}
+
+The agent always connects to the ngrok network via TLS. We support three encryption models based on where TLS is _terminated_:
+
+1. **TLS termination at the ngrok network**. This is the default behavior when using HTTPS tunnels, where ngrok manages TLS certificate generation and renewal for both you and your vendor.
+2. **TLS termination at the ngrok agent**. Often referred to as [zero-knowledge TLS](/docs/tls/tls-termination/#zero-knowledge-tls), this model encrypts your TLS tunnels end-to-end, using filesystem paths to your TLS certificate and key, without requiring you to reconfigure your upstream service for TLS termination.
+3. **TLS termination at your upstream service**. This model implements [end-to-end zero-knowledge encryption](/docs/tls/tls-termination/#zero-knowledge-tls), where the ngrok network forwards unterminated connections through to your upstream application.
+
+### Is my data end-to-end encrypted?
+
+Yes—if you terminate TLS at the ngrok agent or your upstream service using the second or third models listed above.
+
+Contact your vendor if you’re unsure how TLS termination is managed in your site-to-site connectivity architecture.
+
+### Can ngrok see my traffic?
+
+No—if you terminate TLS at the agent or your upstream service.
+
+In all encryption models, the ngrok agent cannot see the traffic it forwards only to your upstream service.
```

Comment:
Was this supposed to read "forward ON to your upstream service"?

In all encryption models, the ngrok agent cannot see the traffic it forwards on to your upstream service.